pkgs/by-name/py/pyflyby/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "pyflyby";
+  version = "1.9.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "deshaw";
+    repo = "pyflyby";
+    rev = version;
+    hash = "sha256-QkoFr9tFtZ+ZEWlxe9csrzoYFl9/V2l4hKYfUWsXUdc=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    six
+    toml
+    isort
+    black
+    ipython
+  ];
+
+  pythonImportsCheck = [ "pyflyby" ];
+
+  meta = with lib; {
+    description = "A set of productivity tools for Python";
+    homepage = "https://github.com/deshaw/pyflyby";
+    license = licenses.mit;
+    maintainers = with maintainers; [ jfvillablanca ];
```

```suggestion
  meta = {
    description = "A set of productivity tools for Python";
    homepage = "https://github.com/deshaw/pyflyby";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ jfvillablanca ];
```
- https://nix.dev/guides/best-practices#with-scopes
- #208242